first_responder.md

```diff
@@ -0,0 +1,121 @@
+# Guidelines for first responder
+
+The DLSS infrastructure team is trying out a new rotating role of "first responder".  PRs are always welcome, but especially while we're ironing things out and having regular retrospectives about the process.  If it sticks, and/or the access team adopts it, we can get rid of this caveat.
+
+## The "first responder" rotation premises
+
+* Respond to production issues in a timely fashion during business hours.
+* Single process for handling questions outside the team's current work cycle.
+* Share the responsibility for production outages across the team in a planned way.
+* Every team member rotates through this responsibility.
+* Encourage cross-training, since a first-responder will likely have to investigate applications with which they are unfamiliar.
+* Shore up missing and outdated documentation of production code and processes for everyone (ops, devops, PSM, stakeholders, devs, etc), informed by actual attempts to find and use said documentation to investigate production issues.
+
+### "First responder" != "on call"
+
+* The idiosyncratic name of the role is intentional.  It is not the duty of the "first responder" to be on-call after hours.  The first responder rotation is an effort to watch and triage production issues in an intentional and organized fashion, since no engineer was officially assigned this responsibility in the past, and such minding of things was haphazard.
+
+## Duties
+
+### Weekly Dependency Updates
+
+#### Ensure dependency update spreadsheet is created
+
+Instructions are here https://docs.google.com/spreadsheets/d/1LysSAPFsRGt9PteWpVswp74xnPXyLLxADpHRfh69VLQ/edit#gid=0
+
+It *may* be that someone in a time zone further east than Palo Alto has already done this, but it is the first responder's responsibility to ensure this gets done.
+
+#### Ensure Monday dependency updates are completed
+
+It is a team task to complete these updates, but the first responder needs to make sure that all codebases needing updates have updates merged and deployed.  Note that some projects need to have PRs created by hand ("Manually clone and update repos that failed bundle update" on line 6 of instructions). It may be helpful to post in channel how many updates each developer should do, given who is working that day and how many PRs there are.
+
+### Verify / Notify coverage for following week
+
+1. Verify first responder for following week is still able to cover it.  Check this on Monday (though it's a good thing to keep in mind throughout the week, in case e.g. unexpected illnesses come up).
+  1. It's the scheduled responder's responsibility to find a swap, not current first responder. _TBD: We'll figure something out if said responder is unavailable due to emergency?_
```

Comment:
hopefully this won't come up often, and won't be too difficult to deal with if it does, but this isn't something we have a solid procedure for right now.